### PR TITLE
Updating Doctrine syntax for getRepository method

### PR DESCRIPTION
**Pull request by meel-io**
Changed Doctrine methods to use the `class` constant syntax which is the prefer syntax now.

### CODE REVIEW

File: doctrine.rst [hunk 1]
```diff
@@ -658,18 +658,18 @@ job is to help you fetch entities of a certain class. You can access the
 repository object for an entity class via::
 
     $repository = $this->getDoctrine()
-        ->getRepository('AppBundle:Product');
+        ->getRepository(Product::class);
 
 .. note::
 
-    The ``AppBundle:Product`` string is a shortcut you can use anywhere
+    You can also use ``AppBundle:Product`` syntax. This string is a shortcut you can use anywhere
```
Comment (meel-io) [inline doctrine.rst]: Not sure about this part

**Comment by javiereguiluz** [comment]
Thanks for this contribution ... but I'm afraid this change must be done in 2.7 branch. I agree with @xabbuh's comment here: https://github.com/symfony/symfony-docs/issues/7892#issuecomment-300433023

**Comment by meel-io** [comment]
Hi! This is my first contribution so let me know of any mistakes. I'll re-create this PR with 2.7 as base branch. I did that originally so I'll just rollback to the original PR if @weaverryan agrees

**Comment by xabbuh** [comment]
Don't worry. We can easily switch the branch while merging (I just tested if there would be any conflicts).

File: doctrine.rst (hunk 1 above)
Comment (javiereguiluz) [inline doctrine.rst]: I think it's OK to leave this little note as a reference of how this was used in the past.

**Review by javiereguiluz — APPROVED** [review]

**Comment by weaverryan** [comment]
I think we need to add a few more `use` statements. We don't need them *everywhere* (sometimes you might have 2 or 3 similar code blocks back-to-back, and so having the `use` on the first one may be enough), but we probably need more. Especially in the `doctrine` chapter.

Other than that, easy win!

**Comment by xabbuh** [comment]
Thank you @meel-io.

**Pull request closed without merging**